test/conftest.py

```diff
@@ -184,27 +186,53 @@ def ferc1_dbf_sql_engine(
     return engine
 
 
-@pytest.fixture(scope="session", name="ferc1_engine_xbrl")
-def ferc1_xbrl_sql_engine(
+@pytest.fixture(scope="session")
+def ferc_xbrl(
     pudl_settings_fixture,
     live_dbs,
     ferc_to_sqlite_settings,
     pudl_datastore_fixture,
 ):
-    """Grab a connection to the FERC Form 1 DB clone.
+    """Extract XBRL filings and produce raw DB+metadata files.
 
-    If we are using the test database, we initialize it from scratch first. If we're
-    using the live database, then we just yield a conneciton to it.
+    Extracts a subset of filings for each form for the year 2021.
     """
     if not live_dbs:
-        pudl.extract.xbrl.xbrl2sqlite(
-            ferc_to_sqlite_settings=ferc_to_sqlite_settings,
-            pudl_settings=pudl_settings_fixture,
-            clobber=False,
-            datastore=pudl_datastore_fixture,
-            workers=5,
-            batch_size=20,
-        )
+        year = 2021
+
+        # Prep datastore
+        datastore = FercXbrlDatastore(pudl_datastore_fixture)
+
+        for form in XbrlFormNumber:
+            raw_archive, taxonomy_entry_point = datastore.get_taxonomy(year, form)
+
+            sqlite_engine = _get_sqlite_engine(form.value, pudl_settings_fixture, True)
+
+            form_settings = ferc_to_sqlite_settings.get_xbrl_dataset_settings(form)
+
+            # Extract every fifth filing
+            filings_subset = datastore.get_filings(year, form)[::5]
```

Comment:
Using every fifth filing. This could be changed, but it seemed like a reasonable balance to me